seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -267,12 +267,12 @@
       "tests": [
         "// Test user did replace var keyword",
         "// Test squaredIntegers is const",
-        "// Test squaredIntegers is an array",
-        "// Test squaredIntegers is [16, 1764, 36]",
-        "// Test function keyword was not used",
-        "// Test arrow => was used",
-        "// Test loop was not used",
-        "// Test map and filter were used"
+        "assert(Array.isArray(squaredIntegers), 'message: squaredIntegers should be an array');",
+        "assert(squaredIntegers[0] === 16 && squaredIntegers[1] === 1764 && squaredIntegers[2] === 36, 'message: squaredIntegers should be [16, 1764, 36]');",
+        "assert(!code.match(/function/g), 'message: function keyword should not be used');",
```

Comment:
@MiloATH This test will not work currently, since the `=>` syntax gets transpiled into the `function` keyword. Before making a PR, the safest thing to do is test each of these locally to make sure they work.